README.md

```diff
@@ -2,6 +2,8 @@
 
 A project based learning activity for people who are getting started with Git and GitHub.
```

Comment:
It should be "a project based on learning activity for people who are getting started with Git and GitHub."